config/repos.sets.yml

```diff
@@ -1,46 +1,46 @@
-  ManageIQ/manageiq-schema:
-  ManageIQ/manageiq-ui-classic:
-  ManageIQ/manageiq-ui-service:
+  xaasio-cmp/xaasio-cmp:
+  xaasio-cmp/xaasio-api:
+  xaasio-cmp/xaasio-automation_engine:
+  xaasio-cmp/xaasio-consumption:
+  xaasio-cmp/xaasio-content:
+  xaasio-cmp/xaasio-decorators:
+  xaasio-cmp/xaasio-gems-pending:
+  xaasio-cmp/xaasio-schema:
+  xaasio-cmp/xaasio-ui-classic:
+  xaasio-cmp/xaasio-ui-service:
 providers: &providers
-  ManageIQ/manageiq-providers-amazon:
-  ManageIQ/manageiq-providers-ansible_tower:
-  ManageIQ/manageiq-providers-autosde:
-  ManageIQ/manageiq-providers-awx:
-  ManageIQ/manageiq-providers-azure:
-  ManageIQ/manageiq-providers-azure_stack:
-  ManageIQ/manageiq-providers-cisco_intersight:
-  ManageIQ/manageiq-providers-embedded_terraform:
-  ManageIQ/manageiq-providers-foreman:
-  ManageIQ/manageiq-providers-google:
-  ManageIQ/manageiq-providers-ibm_cic:
-  ManageIQ/manageiq-providers-ibm_cloud:
-  ManageIQ/manageiq-providers-ibm_power_hmc:
-  ManageIQ/manageiq-providers-ibm_power_vc:
-  ManageIQ/manageiq-providers-ibm_terraform:
-  ManageIQ/manageiq-providers-kubernetes:
-  ManageIQ/manageiq-providers-kubevirt:
-  ManageIQ/manageiq-providers-lenovo:
-  ManageIQ/manageiq-providers-nsxt:
-  ManageIQ/manageiq-providers-nuage:
-  ManageIQ/manageiq-providers-nutanix:
-  ManageIQ/manageiq-providers-openshift:
-  ManageIQ/manageiq-providers-openstack:
-  ManageIQ/manageiq-providers-oracle_cloud:
-  ManageIQ/manageiq-providers-ovirt:
-  ManageIQ/manageiq-providers-red_hat_virtualization:
-  ManageIQ/manageiq-providers-redfish:
-  ManageIQ/manageiq-providers-terraform_enterprise:
-  ManageIQ/manageiq-providers-vmware:
-  ManageIQ/manageiq-providers-workflows:
+  xaasio-cmp/xaasio-providers-amazon:
+  xaasio-cmp/xaasio-providers-ansible_tower:
+  xaasio-cmp/xaasio-providers-autosde:
+  xaasio-cmp/xaasio-providers-awx:
+  xaasio-cmp/xaasio-providers-azure:
+  xaasio-cmp/xaasio-providers-azure_stack:
+  xaasio-cmp/xaasio-providers-cisco_intersight:
+  xaasio-cmp/xaasio-providers-embedded_terraform:
+  xaasio-cmp/xaasio-providers-foreman:
+  xaasio-cmp/xaasio-providers-google:
+  xaasio-cmp/xaasio-providers-ibm_cic:
+  xaasio-cmp/xaasio-providers-ibm_cloud:
+  xaasio-cmp/xaasio-providers-ibm_power_hmc:
+  xaasio-cmp/xaasio-providers-ibm_power_vc:
+  xaasio-cmp/xaasio-providers-ibm_terraform:
+  xaasio-cmp/xaasio-providers-kubernetes:
+  xaasio-cmp/xaasio-providers-kubevirt:
+  xaasio-cmp/xaasio-providers-lenovo:
+  xaasio-cmp/xaasio-providers-nsxt:
+  xaasio-cmp/xaasio-providers-nuage:
+  xaasio-cmp/xaasio-providers-nutanix:
+  xaasio-cmp/xaasio-providers-openshift:
+  xaasio-cmp/xaasio-providers-openstack:
+  xaasio-cmp/xaasio-providers-oracle_cloud:
+  xaasio-cmp/xaasio-providers-ovirt:
+  xaasio-cmp/xaasio-providers-red_hat_virtualization:
+  xaasio-cmp/xaasio-providers-redfish:
+  xaasio-cmp/xaasio-providers-terraform_enterprise:
+  xaasio-cmp/xaasio-providers-vmware:
+  xaasio-cmp/xaasio-providers-workflows:
```

Comment:
you don't need to modify this file.  This is for the bot when we do cross-repo-tests.

However, you might need to have your own set for other purposes.  What you can do there is create your own repo set file like repos.xassio-cmp.yml, and have these alternative repos in it.  Then when you call release_branch or release_tag, you can pass `--repo-set xassio-cmp` and it will use your list instead of the default list.